danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/GitLabKtx.kt

```diff
@@ -0,0 +1,47 @@
+package systems.danger.kotlin
```

Comment:
Also I'm not sure what is the best place to put this. I see `GitLab` object only as a data holder and as a direct mapping of it's api representation so I've placed it to separate file and use it as an extension but I don't know if it follows this project's code style 🤷

Reply:
I'm fine either way, I would personally have used the GitLab file, but no strong opinion, don't know if @gianluz has a strong opinion.

What I'm thinking thought is that this is generic enough to live in `Git` (given it uses git to retrieve the count).
In this way it could be used also from Bitbucket

Reply:
I think that `Git` would be ideal place as well but I've used `GitLab` because it provides me metadata about the commits which I need to pass to a `git` command. I have not found any way how to access this info from other place. Do you have any suggestion?

Reply:
Ok, I might have a hint. I can see that in the JSON from danger-js is also list of commits, that is not parsed currently. I can run `git diff --numstats` for this commits and aggregate changed lines based on that (I think that is the similar approach that danger-js has). I will investigate more after later today :)

Reply:
i'm happy to use separate files for the extensions, we just need to keep conventions once started.. eg: `GitHubKtx.kt` `GitKtx.kt` ...

Reply:
Ok guys, take a look now (I've force pushed for cleaner git history). I've basically took the provided list of commits from the JSON from danger-js and I've computed this statistics for each of the commit and then just sum it up. It works when I try to test it on my GitLab MRs. I've also made it as extensions over `Git` which is more generic